sql/expression/aggregation_test.go

```diff
@@ -96,3 +97,53 @@ func TestFirst_Eval(t *testing.T) {
 	c.Merge(b, b2)
 	assert.Equal(int32(1), c.Eval(b))
 }
+
+func TestMax_Name(t *testing.T) {
+	assert := require.New(t)
+
+	m := NewMax(NewGetField(0, sql.Int32, "field", true))
+	assert.Equal("max(field)", m.Name())
+}
+
+func TestMax_Eval_Int32(t *testing.T) {
```

Comment:
Can you add some tests with nulls involved? thew must be ignored. Also, check the corner cases on MAX here:

http://www-cs-students.stanford.edu/~wlam/compsci/sqlnulls

Reply:
@ajnavarro since the NULL thing is an enhancement to be implemented #33, can I merge this PR?

Reply:
Even if nulls are not implemented per se, we can start handling nulls inside every node we implement.